modules/transactionpool/subscribe.go

```diff
@@ -0,0 +1,46 @@
+package transactionpool
+
+import (
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// updateSubscribersTransactions sends a new transaction pool update to all
+// subscribers.
+func (tp *TransactionPool) updateSubscribersTransactions() {
+	var txns []types.Transaction
+	var cc modules.ConsensusChange
+	for _, tSet := range tp.transactionSets {
+		txns = append(txns, tSet...)
+	}
+	for _, tSetDiff := range tp.transactionSetDiffs {
+		cc = cc.Append(tSetDiff)
+	}
+	for _, subscriber := range tp.subscribers {
+		subscriber.ReceiveUpdatedUnconfirmedTransactions(txns, cc)
+	}
+}
+
+// updateSubscribersConsensus sends a new consensus change to all subscribers.
+func (tp *TransactionPool) updateSubscribersConsensus(cc modules.ConsensusChange) {
+	for _, subscriber := range tp.subscribers {
+		subscriber.ReceiveConsensusSetUpdate(cc)
+	}
+}
```

Comment:
so the wallet subscribes to the tpool but not the consensusset, and receives consensus changes through the tpool? I suppose that makes sense; it keeps the two updates synchronized.